src/lib/io/Kconfig

```diff
@@ -81,3 +81,12 @@ config USE_MEMMAP
         Users must provide a mapping - for instance, using a JSON
         file - that will be used to define where on memory
         entries will be saved. Examples of memory are NVRAM or EEPROM.
+
+config USE_IPM
+    bool "Inter-processor messaging"
+    depends on ZEPHYR
```

Comment:
@ibriano what's your take on this? `depends on ZEPHYR` or declare a feature that only zephyr defines?